RELEASE-NOTES.md

```diff
@@ -129,6 +129,9 @@ This includes API changes we did not warn about since at least `3.11.0` (2021-01
 - `softmax` and `log_softmax` functions added to `math` module (see [#5279](https://github.com/pymc-devs/pymc/pull/5279)).
 - ...
 
+### Maintenance
```

Comment:
Upon reconsideration, we probably don't need to mention it at all because it doesn't add new functionality, we just need re move it from the list of not-working things.

Reply:
Probably fine to leave it under maintenance. Users likely will not notice something missing from the non-working list but will notice it here, and VI is a pretty big feature to have working again.

Reply:
Oops, I pushed a commit that removed this change. I do agree with @twiecki, this changelist should reflect the difference between current and v3, and as such it does not make sense to say something is still working...?